src/main/resources/logback-spring.xml

```diff
@@ -9,7 +9,7 @@
         <encoder>
             <!-- %ex{full, SECURITY_STACKTRACE_EVAL} will display the full stack trace of the exception only
              if the evaluator called SECURITY_STACKTRACE_EVAL returns false. -->
-            <pattern>%d{ISO8601} [thread=%thread] [%-5p] [%c] %X{user}- %m%n%ex{full, SECURITY_STACKTRACE_EVAL}</pattern>
+            <pattern>%d{ISO8601} [thread=%thread] [%-5p] [%c] %X{user}- %M:%line %m%n%ex{full, SECURITY_STACKTRACE_EVAL}</pattern>
```

Comment:
For me, I find that logging the method and line number in a standard log line just clutters the log. We already put the class name in there which makes the line long enough.

For a change like this one, I'd suggest that you poll the team when they get back since this is only MY opinion and others may like it! :)

Here's a sample of the log with this formatting change applied so that others can take a look at it and see what they think.

```
2020-11-25 12:58:22,857 [thread=main] [INFO ] [org.candlepin.subscriptions.BootApplication] - logStarting:55 Starting BootApplication on redcanoe with PID 2632544 (/home/mstead/sandbox/swatch/subscriptions/build/classes/java/main started by mstead in /home/mstead/sandbox/swatch/subscriptions)
2020-11-25 12:58:22,861 [thread=main] [INFO ] [org.candlepin.subscriptions.BootApplication] - logStartupProfileInfo:655 The following profiles are active: worker,kafka-queue
2020-11-25 12:58:23,919 [thread=main] [WARN ] [org.jboss.resteasy.springboot.ResteasyEmbeddedServletInitializer] - postProcessBeanFactory:249 No JAX-RS resource Spring beans have been found
2020-11-25 12:58:23,919 [thread=main] [WARN ] [org.jboss.resteasy.springboot.ResteasyEmbeddedServletInitializer] - postProcessBeanFactory:261 JAX-RS Application class org.candlepin.subscriptions.resteasy.JaxrsApplication has no ApplicationPath annotation, so it will not be registered
2020-11-25 12:58:28,032 [thread=main] [INFO ] [org.candlepin.subscriptions.cloudigrade.ConcurrentApiFactory] - getObject:54 Cloudigrade service URL: http://localhost:8080/api/cloudigrade/v2
2020-11-25 12:58:28,460 [thread=main] [INFO ] [org.candlepin.subscriptions.task.queue.kafka.KafkaTaskQueue] - <init>:43 Creating Kafka task queue...
2020-11-25 12:58:28,654 [thread=main] [WARN ] [org.springframework.boot.autoconfigure.orm.jpa.JpaBaseConfiguration$JpaWebConfiguration] - openEntityManagerInViewInterceptor:220 spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-11-25 12:58:28,705 [thread=main] [WARN ] [org.candlepin.subscriptions.rbac.RbacApiFactory] - getObject:59 RBAC service URL not set...

```

Reply:
One thought here is that if others tend to like this format, perhaps we should move it into the class [ ] grouping as follows.

```
<pattern>%d{ISO8601} [thread=%thread] [%-5p] [%c %M:%line] %X{user}- %m%n%ex{full, SECURITY_STACKTRACE_EVAL}</pattern>
```

Reply:
I can see that.  Personally, I'm a huge fan of the method name/line number being included in at least the console logs and use them all the time - but agreed, totally a preference thing - I can just gitignore it or something to keep it there locally if you guys aren't a fan.  Not sure if it changes anyone's opinion, but I've configured log4j2 in the past to abbreviate the package names leading up to the class name to cut down on clutter.  Not sure if logback has the same capability or not.

Reply:
Re abbreviating package names: [It does](http://logback.qos.ch/manual/layouts.html#conversionWord)

My only hesitation around adding the method names and line numbers is a warning in the docs "Generating the method name is not particularly fast. Thus, its use should be avoided unless execution speed is not an issue."  There's a similar warning for line numbers.

I'd be interested to see benchmarks on it.  But I don't want to add it without at least checking how it affects performance.

Reply:
I reverted this commit for the time being